libsolidity/ast/Types.h

```diff
@@ -46,8 +46,10 @@ namespace solidity::frontend
 class TypeProvider;
 class Type; // forward
 class FunctionType; // forward
+class ModifierType; // forward
 using TypePointer = Type const*;
 using FunctionTypePointer = FunctionType const*;
+using ModifierTypePointer = ModifierType const*;
```

Comment:
Previously, there was a reason in using the `...Pointer` aliases, but now the word is even longer than the explicit type.